website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Connect Microsoft Fabric"
+description: "Configure Microsoft Fabric connection."
+sidebar_label: "Connect Microsoft Fabric"
+---
+
+The following are the required fields for setting up a connection with a [Microsoft Fabric](https://docs.starburst.io/starburst-enterprise/index.html) using service principal authentication. 
+
+| Field | Description |
+| --- | --- |
+| **Server** | The service principal's **host** value for the Fabric test endpoint. |
+| **Port** | The port to connect to Microsoft Fabric. By default, it's 1433 for the standard SQL server port number. |
+| **Database** | The service principal's **database** value for the Fabric test endpoint. |
+| **Authentication** | Choose **Service Principal** from the dropdown. | 
+| **Tenant ID** | The service principal's **Directory (tenant) ID**. |
+| **Client ID** | The service principal's **application (client) ID id**. |
+| **Client secret** | The service principal's **client secret** (not the **client secret id**). |  
```

Comment:
can you somehow show that its either these fields, OR the AD user and password fields?

Reply:
great flag! it'd be great if we can add this info to the docs page. do we have username/password credentials that i can play around with? i'd be happy to add those required fields to the new connect page